pallets/runtime/common/src/bridge.rs

```diff
@@ -425,83 +426,110 @@ decl_module! {
         }
 
         /// Forces handling a transaction by bypassing the bridge limit and timelock.
-        #[weight = SimpleDispatchInfo::FixedOperational(20_000)]
+        #[weight = SimpleDispatchInfo::FixedOperational(250_000)]
         pub fn force_handle_bridge_tx(origin, bridge_tx: BridgeTx<T::AccountId, T::Balance>) -> DispatchResult {
             // NB: To avoid code duplication, this uses a hacky approach of temporarily whitelisting the did
             let sender = ensure_signed(origin)?;
-            ensure!(sender == Self::admin(), Error::<T>::BadAdmin);
-            if let Some(did) = T::CddChecker::get_key_cdd_did(&AccountKey::try_from(bridge_tx.recipient.clone().encode())?) {
-                if !Self::bridge_whitelist(did) {
-                    // Whitelist the did temporarily
-                    <BridgeLimitWhitelist>::insert(did, true);
-                    Self::handle_bridge_tx_now(bridge_tx, false)?;
-                    <BridgeLimitWhitelist>::insert(did, false);
-                } else {
-                    // Already whitelisted
-                    return Self::handle_bridge_tx_now(bridge_tx, false);
-                }
-            } else {
-                return Err(Error::<T>::NoValidCdd.into());
+            Self::force_handle_signed_bridge_tx(&sender, bridge_tx)
+        }
+
+        /// Forces handling a vector of transactions by bypassing the bridge limit and timelock.
+        /// The vector is processed until the first proposal which causes an error, in which case
+        /// the error is returned and the rest of proposals are not processed.
+        ///
+        /// # Weight
+        /// `50_000 + 200_000 * bridge_txs.len()`
+        #[weight = FunctionOf(
+            |(bridge_txs,): (
+                &Vec<BridgeTx<T::AccountId, T::Balance>>,
+            )| {
+                50_000 + 200_000 * u32::try_from(bridge_txs.len()).unwrap_or_default()
+            },
+            DispatchClass::Operational,
+            true
+        )]
+        pub fn force_handle_bridge_txs(origin, bridge_txs: Vec<BridgeTx<T::AccountId, T::Balance>>) ->
+            DispatchResult
+        {
+            let sender = ensure_signed(origin)?;
+            for tx in bridge_txs {
+                Self::handle_signed_bridge_tx(&sender, tx)?;
             }
             Ok(())
         }
 
         /// Proposes a bridge transaction, which amounts to making a multisig proposal for the
         /// bridge transaction if the transaction is new or approving an existing proposal if the
         /// transaction has already been proposed.
-        #[weight = SimpleDispatchInfo::FixedOperational(50_000)]
+        #[weight = SimpleDispatchInfo::FixedOperational(800_000)]
```

Comment:
What was the reason to increase the operational transaction fee?

Reply:
I think the weight of `propose*` extrinsics should be greater than the weight of `create_or_approve_proposal_as_key` in the multisig module, which is 750000. This is because they call the same function `create_or_approve_proposal`, yet the bridge propose extrinsics additionally create the proposal value.